magma/frontend/verilog.py

```diff
@@ -1,89 +1,138 @@
-from __future__ import absolute_import
-from __future__ import print_function
-import warnings
-
 from mako.template import Template
-
 from ..circuit import DeclareCircuit, DefineCircuit, EndDefine
-
+from ..common import deprecated
+from ..logging import root_logger
 from .pyverilog_importer import PyverilogImporter
 from .verilog_importer import ImportMode
 
-from ..logging import root_logger
 
 _logger = root_logger().getChild('from_verilog')
 
-
-__all__  = ['DeclareFromVerilog']
+__all__ = ['DeclareFromVerilog']
 __all__ += ['DeclareFromVerilogFile']
 __all__ += ['DeclareFromTemplatedVerilog']
 __all__ += ['DeclareFromTemplatedVerilogFile']
 __all__ += ['DefineFromVerilog']
 __all__ += ['DefineFromVerilogFile']
 __all__ += ['DefineFromTemplatedVerilog']
 __all__ += ['DefineFromTemplatedVerilogFile']
+__all__ += ['declare_from_verilog']
+__all__ += ['declare_from_verilog_file']
+__all__ += ['define_from_verilog']
+__all__ += ['define_from_verilog_file']
```

Comment:
While we're at it, should we transition to having `__init__` import the desired functions, rather than using `__all__` and `import *`?

Reply:
Agreed, will do.